### PR TITLE
Add support python 3.8

### PR DESCRIPTION
**Pull request by popix144**
[x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-python/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Adding support for Python 3.8. Changes in the ast module have been taken into account (ast.Num, ast.Str, ast.NameConstant became ast.Constant).

### Why should this Pull Request be merged?

It offers support for the newest major release of Python.

### What testing has been done?

All tests run with tox passed.

### CODE REVIEW

File: src/niveristand/_translation/utils.py [hunk 1]
```diff
@@ -92,8 +115,24 @@ def get_variable_name_from_node(node):
...
+    if sys.version_info < (3, 8) and type(node) is ast.Str:
         channel_name = node.s
+    elif check_ast_constant_str(node):
+        channel_name = node.value
     else:
         raise errors.TranslateError(_errormessages.invalid_type_for_channel_ref)
     return channel_name
+
+
+def check_ast_constant_str(node):
```
Comment (arnoldcsorvasi) [inline src/niveristand/_translation/utils.py:127]: Please modify these check_ast_constant_* methods to do the version checking as well and use them above to check if a node is of that type.E.g.

```
def is_node_ast_str(node):
    if sys.version_info < (3, 8) and type(node) is ast.Str or isinstance(node, ast.Constant) and isinstance(node.value, str):
        return true
    else:
        return false
```

or something similar. Also please create methods to extract the value from the node based on its type. E.g.

```
def get_value_from_str_node(node):
    if sys.version_info < (3, 8):
        return node.s
    else:
        return node.value
```
or something similar. If it makes more sense you can combine them, but I think having clear methods to extract the value from different AST nodes will make it easier to understand.

**Review by arnoldcsorvasi — CHANGES_REQUESTED** [review]

File: src/niveristand/_translation/utils.py [hunk 2]
```diff
@@ -26,54 +27,58 @@ def get_value_from_node(node, resources):
...
                 datavalue = get_element_value(node.args[0])
             elif type(node.args[0]) is ast.Name:
                 if node.args[0].id in symbols._symbols:
                     datavalue = symbols._symbols[node.args[0].id]
                 else:
                     raise TranslateError(_errormessages.init_var_invalid_type)
-            elif type(node.args[0]) is ast.NameConstant:
-                if node.args[0].value is True or node.args[0].value is False:
-                    datavalue = node.args[0].value
+            # In Python 3.8, NameConstant is Constant
```
Comment (arnoldcsorvasi) [inline src/niveristand/_translation/utils.py:37]: Extra comment.

**Review by arnoldcsorvasi — COMMENTED** [review]
The unittests folder should move from src to tests.

Please fix these flake errors:
src/niveristand/_translation\py2rtseq\call_transformer.py:2:1: F401 'sys' imported but unused
src/niveristand/_translation\py2rtseq\transformers.py:65:84: W292 no newline at end of file
src/niveristand/_translation\py2rtseq\try_transformer.py:2:1: F401 'sys' imported but unused
tests/test_multitask.py:295:9: F811 redefinition of unused 'f1' from line 291

File: src/niveristand/_translation/utils.py (hunk 2 above)
Comment (popix144) [reply src/niveristand/_translation/utils.py:37]: image

In tests/test_multitask.py, it says that redefinition is exactly what is tested, so I guess I should not make any changes in that file.

**Comment by popix144** [comment]
> The unittests folder should move from src to tests.
> 
> Please fix these flake errors:
> src/niveristand/_translation\py2rtseq\call_transformer.py:2:1: F401 'sys' imported but unused
> src/niveristand/_translation\py2rtseq\transformers.py:65:84: W292 no newline at end of file
> src/niveristand/_translation\py2rtseq\try_transformer.py:2:1: F401 'sys' imported but unused
> tests/test_multitask.py:295:9: F811 redefinition of unused 'f1' from line 291

**Review by arnoldcsorvasi — APPROVED** [review]

**Review by cheloizaguirre — APPROVED** [review]

**Pull request merged**